pubsublite/admin/pubsublite_admin_test.go

```diff
@@ -0,0 +1,473 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package admin
```

Comment:
These tests are currently set up to test each sample completely independently, which is OK. What do you think of combining a few (or all?) of the tests into a single test? For example, you can create a topic, modify it, and delete it, all using samples, in a single test. Can we extend that idea to ~all of the samples?

A benefit of that is there is less setup and teardown that can lead to flakiness when the tests run.